data/saas/config/slack_config.yml

```diff
@@ -0,0 +1,41 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Slack SaaS Config
+  type: slack
+  description: A sample schema representing the Slack connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: access_token
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: bearer
+      configuration:
+        token: <access_token>
+
+  test_request:
+    method: GET
+    path: /api/api.test
+    query_params:
+      - name: email
+        value: test@test.com
+
+
+  endpoints:
+    - name: user
```

Comment:
for the new connector, we planned to access the User directly and manipulate the User object instead of the User's message.
When we delete a user using [Admin API](https://api.slack.com/methods/admin.users.remove), it automatically removes the associated messages.

Reply:
@Zeeshan-Ethyca that sounds good in terms of erasures/updates, but we should still give back the message data associated with a given user as part of the access request. let me know if you've got further questions on it!